doc/api/fs.md

```diff
@@ -378,7 +378,8 @@ fs.appendFile('message.txt', 'data to append', 'utf8', callback);
 
 Any specified file descriptor has to have been opened for appending.
 
-_Note: Specified file descriptors will not be closed automatically._
+_Note: If a file descriptor is specified as the `file`, it will not be closed 
+automatically._
```

Comment:
I’m +1 on keeping it, too. The file descriptor obviously should be closed at some point, that just doesn’t happen by calling `{append,read,write}File`.